layouts/partials/hubspot-form.html

```diff
@@ -0,0 +1,9 @@
+<script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
+<script>
+    hbspt.forms.create({
+        portalId: "4429525",
+        formId: {{ .hubspotFormID }},
+        css: "",
+        cssClass: "{{ .classNames }}",
```

Comment:
What happens when `classNames` isn't specified in the `dict` passed to the partial? Will it just be empty as expected?

Reply:
Yep.